src/lightning/pytorch/utilities/imports.py

```diff
@@ -35,21 +35,25 @@ def _try_import_module(module_name: str) -> bool:
     try:
         __import__(module_name)
         return True
-    # added also AttributeError fro case of impoerts like pl.LightningModule
+    # Also on AttributeError for failed imports like pl.LightningModule
     except (ImportError, AttributeError) as err:
-        rank_zero_warn(f"Import of {module_name} package failed for some compatibility issues: \n{err}")
+        rank_zero_warn(f"Import of {module_name} package failed for some compatibility issues:\n{err}")
         return False
 
 
-@functools.lru_cache(maxsize=1)
```

Comment:
why are you removing this?

Reply:
`RequirementCache` is already cached (duh!) and `_try_import_module` already has a `lru_cache` annotation